java-checks/src/main/java/org/sonar/java/checks/SerializableFieldInSerializableClassCheck.java

```diff
@@ -129,7 +143,7 @@ private static boolean implementsSerializable(@Nullable Type type) {
       return implementsSerializable(((Type.ArrayType) type).elementType());
     }
     if (type.isClass() || ((JavaType) type).isTagged(JavaType.TYPEVAR)) {
-      return type.erasure().isSubtypeOf("java.io.Serializable");
+      return type.erasure().isSubtypeOf(Serializable.class.getName());
```

Comment:
This has nothing to do with your contribution. I would keep the class name as a String as it is clearer and avoid to import Serializable which is not required. Same remark for the other arguments.